megalinter/Linter.py

```diff
@@ -1225,7 +1225,9 @@ def build_lint_command(self, file=None) -> list:
                 self.final_config_file = self.final_config_file.replace(
                     self.workspace, DEFAULT_DOCKER_WORKSPACE_DIR
                 )
-            if self.cli_config_arg_name.endswith("="):
+            if self.cli_config_arg_name.endswith(
+                "="
+            ) or self.cli_config_arg_name.endswith(":"):
```

Comment:
Okay I actually think I found a fix to the clang-format config arg issue here. Take a look and see if it makes sense.

I'm just not sure how to accurately test that this somehow doesn't break another linter.

Reply:
CI jobs should tell you :)
It seems safe to me, as only descriptors with cli_config_arg_name ending by `:` should be impacted

Reply:
In case it is too complicated, you can always define a python class that will override default methods :)